javascript/operators/await.json

```diff
@@ -79,24 +79,10 @@
                 "version_added": false
               },
               "firefox": {
-                "version_added": false,
-                "notes": "Available only in nightly builds. See <a href='https://bugzil.la/1681046'>bug 1681046</a>.",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "javascript.options.experimental.top_level_await"
-                  }
-                ]
+                "version_added": "89"
```

Comment:
Nice cleanup! Ordinarily, I'd suggest keeping old prefs and adding new support statements, but this is a weird one with Nightly-only.